Update temlate CSS

Dgikar opened the pull request:
Pull Request for Issue # 9449 (https://github.com/joomla/joomla-cms/pull/9449/commits/dc1cca1c29d2bdfea3c9c25eeaf0eb699c278d39)
## The language icons will be displayed in a column as now, but in the line that saves space, allowing you to display much more items on the screen.

The same in Russian:
Значки языков, будут отображаться не в столбик как сейчас, а в строку, что СУЩЕСТВЕННО экономит пространство, позволяя отобразить намного больше пунктов на экране.

![015](https://cloud.githubusercontent.com/assets/4268737/13824519/089a776a-ebb7-11e5-913e-6644ad2416dc.jpg)

Dgikar committed:
Update temlate CSS

The language icons will be displayed in a column as now, but in the line that saves space, allowing you to display much more items on the screen.
----------------------------------------------------------------
The same in Russian:
Значки языков, будут отображаться не в столбик как сейчас, а в строку, что СУЩЕСТВЕННО экономит пространство, позволяя отобразить намного больше пунктов на экране.

brianteeman commented:
Please update the less file and run generatesss.php and not edit the CSS file directly<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9449">issues.joomla.org/joomla-cms/9449</a>.</sub>

Dgikar commented:
Sorry, but I can't...

zero-24 commented:
@dgikar

Edit this file: https://github.com/Dgikar/joomla-cms/blob/patch-1/administrator/templates/isis/less/template.less

Than the changes get added to this PR ;)

Dgikar committed:
Update temlate CSS #9449

The language icons will be displayed in a column as now, but in the line that saves space, allowing you to display much more items on the screen.
----------------------------------------------------------------
The same in Russian:
Значки языков, будут отображаться не в столбик как сейчас, а в строку, что СУЩЕСТВЕННО экономит пространство, позволяя отобразить намного больше пунктов на экране.

Dgikar commented:
@zero-24 
Done

infograf768 commented:
In staging It does not display in one vertical row but 2. No idea how you get that.
![screen shot 2016-03-17 at 10 19 13](https://cloud.githubusercontent.com/assets/869724/13841557/e25dc930-ec29-11e5-9b78-87f306bff04b.png)

Putting them in one horizontal row would force large screens. 
I am not in favour of this change.

brianteeman commented:
I can confirm the screenshot of @infograf768 above in the current staging. Changing from this to a single row is not an improvement.

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9449">issues.joomla.org/joomla-cms/9449</a>.</sub>

brianteeman commented:
Closed<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9449">issues.joomla.org/joomla-cms/9449</a>.</sub>

Dgikar commented:
@infograf768 Sorry, but you don't have one line - two line! It's a lot. In **ALL**! my Joomla (v. 1.5, 1.7 ... v. 3.5 rc 4), I (and not only me) have thes problem ewery time!
